tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Azure/AzureServiceBusTests.cs

```diff
@@ -0,0 +1,93 @@
+// <copyright file="AzureServiceBusTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Datadog.Trace.ClrProfiler.IntegrationTests.Helpers;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.Azure
+{
+    // [Trait("RequiresDockerDependency", "true")]
+    [UsesVerify]
+    public class AzureServiceBusTests : TracingIntegrationTest
+    {
+        public AzureServiceBusTests(ITestOutputHelper output)
+            : base("AzureServiceBus", output)
+        {
+        }
+
+        public static IEnumerable<object[]> GetEnabledConfig()
+            => from packageVersionArray in new string[] { string.Empty }
+               from metadataSchemaVersion in new[] { "v0", "v1" }
+               select new string[] { packageVersionArray, metadataSchemaVersion };
+
+        public override Result ValidateIntegrationSpan(MockSpan span, string metadataSchemaVersion) => span.Tags["span.kind"] switch
+        {
+            SpanKinds.Consumer => span.IsAzureServiceBusInbound(metadataSchemaVersion),
+            SpanKinds.Producer => span.IsAzureServiceBusOutbound(metadataSchemaVersion),
+            SpanKinds.Client => span.IsAzureServiceBusRequest(metadataSchemaVersion),
+            _ => throw new ArgumentException($"span.Tags[\"span.kind\"] is not a supported value for the AWS SQS integration: {span.Tags["span.kind"]}", nameof(span)),
+        };
+
+        [SkippableTheory(Skip = "We are unable to test all the features of Azure Service Bus with an emulator. For now, run only locally with a connection string to a live Azure Service Bus namespace")]
```

Comment:
If we're not running it in CI, maybe we should skip building the samples in CI too (`CompileSamplesWindows`, and `CompileSamplesLinuxOrOsx`)? Otherwise it just increases CI team with no real benefit? We still take the hit from the Restore, but I think we just live with that one.

Reply:
Added a project skip in 46542c7eccdecdbb612460fde446cbc4ce3c2cdf . Hopefully it works as intended or I'll have to go back and fix it up 😆

Reply:
I don't see `Samples.AzureServiceBus` in the `integration_tests_windows` build step or the `integration_tests_linux` build step so it looks like I've successfully excluded the project build 🎉